subscriptions/tasks.py

```diff
@@ -412,6 +378,41 @@ def pre_send_process(subscription_id, resend_id=None):
     return context
 
 
+@app.task(
+    autoretry_for=(HTTPError, ConnectionError, Timeout, HTTPServiceError),
+    retry_backoff=True,
+    retry_jitter=True,
+    max_retries=15,
+    acks_late=True,
+    time_limit=10,
+    base=BaseSendMessage
+)
+def get_identity_address(context):
+    if "error" in context:
+        return context
```

Comment:
Not for this PR, but I wonder if we can somehow add this error checking to the base class?

Also, I wonder if this will do the same thing, instead of us having to check the context for errors all the time: http://docs.celeryproject.org/en/latest/userguide/tasks.html#ignore

Reply:
I tried raising `Ignore` but it still just continues to the next task. I'll look into doing this in a better way, haven't found anything yet.